src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -45,7 +45,15 @@ public class CompositeWorkItem : WorkItem
         private TestCommand _setupCommand;
         private TestCommand _teardownCommand;
 
-        private List<WorkItem> _children;
+        /// <summary>
+        /// List of Child WorkItems
+        /// </summary>
+        public List<WorkItem> Children;
```

Comment:
If we are going to promote this to public, it should be made a property with a private setter and change the type to `IList`. Can it be made internal?